docs/dashboards/data-handling.md

```diff
@@ -55,6 +55,43 @@ dataPool: {
 }
 ```
 
+```js
+dataPool: {
+    connectors: [{
+        id: 'my-json-connector',
+        type: 'JSON',
+        options: {
+            dataUrl: 'https://demo-live-data.highcharts.com/instance-details.json',
+        }
+    }]
+}
+```
+
+```js
+dataPool: {
+    connectors: [{
+        id: 'my-google-sheets-connector',
+            type: 'GoogleSheets',
+            options: {
+                googleAPIKey: 'AIzaSyCQ0Jh8OFRShXam8adBbBcctlbeeA-qJOk',
+                googleSpreadsheetKey: '1U17c4GljMWpgk1bcTvUzIuWT8vdOnlCBHTm5S8Jh8tw'
```

Comment:
This should not be valid, but have in the middle most characters replaced with `...`.